docs/developers/func-file.md

```diff
@@ -1,23 +1,23 @@
-# Function files
+# Func files
 
-Functions files are used to assist fn to help you when creating functions.
+Func files are used to assist the fn-project to help you when creating functions.
 
-An example of a function file:
+An example of a func file:
 
 ```yaml
+schema_version: 20180708
```

Comment:
2018-07-08? Easier to read and more obvious as to meaning.  Do we need days? As in, would 2018-07 be enough?

Reply:
I think we do need days, lower would probably be pushing it.

Format wise, this is normal and is pretty obvious. It doesn't really need to be read easily as a date, that's not really it's function.